### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by haddadzineddine**
Improve code quality by:

1- delete unnecessary comments (code should be self-descriptive)
2- apply soc principle (separation do concern) by introducing a new functions / controllers ...
3- fix the UI
4- improve code style using PHP-cs-fixer (with laravel pint)

### CODE REVIEW

File: resources/views/components/search-blood-form.blade.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
                     <select name="wilaya" id="wilayaSelect" class="form-select form-select-lg" required>
                         <option selected hidden style="display:none" value="">{{ __('homePage.wilaya') }}</option>
                         @foreach ($wilayas as $wilaya)
-                            <option value="{{ $wilaya['id'] }}">{{ $wilaya['id'].'. '.$wilaya[$wilayaName] }} </option>
+                            <option value="{{ $wilaya['id'] }}">{{ $wilaya['id'].'. '.$wilaya['name'] }} </option>
```
Comment (dev-dev-15) [inline resources/views/components/search-blood-form.blade.php:29]: $wilaya['name'] will always return the name in Ascii, not in the correct language, let it be $wilaya['nameName'], that last will return the name in the correct language, please have a look at app/View/Components/SearchBloodForm.php line 322

File: resources/views/pages/auth/register.blade.php [hunk 2]
```diff
@@ -40,16 +40,13 @@
...
-                    <option selected hidden style="display:none" value="">{{ __('registerPage.bloodGroup') }}</option>
-                    <option value="1">A+</option>
-                    <option value="2">A-</option>
-                    <option value="3">B+</option>
-                    <option value="4">B-</option>
-                    <option value="5">O+</option>
-                    <option value="6">O-</option>
-                    <option value="7">AB+</option>
-                    <option value="8">AB-</option>
+                <select data-validator-func="bloodGroupValidator" name="blood_group" class="form-select "
```
Comment (dev-dev-15) [inline resources/views/pages/auth/register.blade.php:43]: remove the space in class="form-select "

File: resources/views/components/search-blood-form.blade.php (hunk 1 above)
Comment (haddadzineddine) [reply resources/views/components/search-blood-form.blade.php:29]: Yes , i'm already aware of that , check out the **Wilaya Model**  ( i created accessor for name attribute )

**Review by haddadzineddine — COMMENTED** [review]
i didn't notice that , u're right

File: resources/views/components/search-blood-form.blade.php (hunk 1 above)
Comment (haddadzineddine) [reply resources/views/components/search-blood-form.blade.php:29]: thanks , i didn't notice that

**Pull request merged**